proto/p2p/p2p.proto

```diff
@@ -415,8 +415,8 @@ message AppRequestFailed {
   bytes chain_id = 1;
   // Request id of the original AppRequest
   uint32 request_id = 2;
-  // VM defined error code
-  uint32 error_code = 3;
+  // VM defined error code. VMs may define error codes > 0.
```

Comment:
Using `sint32` instead of `int32` since the protobuf docs say that negative numbers are less efficent:
```
Uses variable-length encoding. Inefficient for encoding negative numbers – if your field is likely to have negative values, use sint32 instead.
--



```